doc/getting-started/building-the-node-using-nix.md

```diff
@@ -44,6 +44,10 @@ or run in in one go:
 nix run github:input-output-hk/cardano-node#mainnet/node
 ```
 
+To build the cardano-node executable:
```

Comment:
I would have reworded this to underscore -- something like -- "to build just the cardano-node executable"

Reply:
And it doesn't help that line 35 above doesn't really explain what it achieves.

We could reword it too, from:
> Once Nix is installed, log out and then log back in.
>
> After that you can build the full node package with Mainnet configuration as follows:

Reply:
Amended the commit with your suggestions, thanks Serge!

Reply:
There goes another amendment! =)